docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -23,14 +23,38 @@ table lists the returned columns and what statistics they represent. Any
 additional statistics collected on the data source, other than those listed
 here, are not included.
 
-==========================  ============================================================= =================================
-Column                      Description                                                   Notes
-==========================  ============================================================= =================================
-``column_name``             The name of the column                                        ``NULL`` in the table summary row
-``data_size``               The total size in bytes of all of the values in the column    ``NULL`` in the table summary row. Available for columns of textual types (``CHAR``, ``VARCHAR``, etc)
-``distinct_values_count``   The estimated number of distinct values in the column m       ``NULL`` in the table summary row
-``nulls_fractions``         The portion of the values in the column that are ``NULL``     ``NULL`` in the table summary row.
-``row_count``               The estimated number of rows in the table                     ``NULL`` in column statistic rows
-``low_value``               The lowest value found in this column                         ``NULL`` in the table summary row. Available for columns of numeric types (``BIGINT``, ``DECIMAL``, etc)
-``high_value``              The highest value found in this column                        ``NULL`` in the table summary row. Available for columns of numeric types (``BIGINT``, ``DECIMAL``, etc)
-==========================  ============================================================= =================================
+.. list-table:: Statistics
+  :widths: 20, 40, 40
+  :header-rows: 1
+
+  * - Column
+    - Description
+    - Notes
+  * - ``column_name``
+    - The name of the column
+    - ``NULL`` in the table summary row
+  * - ``data_size``
+    - The total size in bytes of all of the values in the column
+    - ``NULL`` in the table summary row. Available for columns of textual types
+      (``CHAR``, ``VARCHAR``, etc)
```

Comment:
Should we change this to string data types? https://trino.io/docs/current/language/types.html#string .. or are others like MAP or so also supported?

Reply:
string looks fine but I don't know if stats for `VARBINARY` is supported or not. Someone needs to verify.

Reply:
Who would know?

Reply:
We don't show data_size for fixed width types, it should show up for any variable width type (provided connector supports it, I think most connectors won't give stats for structural types).